src/pages/_app.js

```diff
@@ -1,5 +1,11 @@
-import '@/styles/globals.css'
+import { GlobalStyles } from "../Styles/Styles";
```

Comment:
```suggestion
import GlobalStyles from "../Styles/Styles.js";
```
It's a default export on the other side. Hence, you're not allowed to use a named import here.